lib/tictactoe/game.rb

```diff
@@ -1,9 +1,55 @@
+require "./lib/tictactoe/board"
 
 module Tictactoe
-    print "Welcome to Tic-Tac-Toe"
-    
-    class Game 
+  print "Welcome to Tic-Tac-Toe"
+
+  class Game
+    attr_reader :board
+
+    def initialize
+      @symbols = []
+      @board = Board.new
+    end
+
+    def draw
+      @board.draw
+    end
+
+    def check_position(pos)
+      if @board.positions[pos] != "-"
+        raise "Error"
+      end
+      return true
     end
-    class Board 
+
+    def play(move, symbol)
+      @symbols.push(symbol)
+      if @symbols[-2] and @symbols[-1] and @symbols[-2] == @symbols[-1]
```

Comment:
Looks like you are accounting for a scenario where @symbols[-2] or @symbols[-1] is `nil`. Let's write a test for it!